app/services/applications/revert_to_pending.rb

```diff
@@ -0,0 +1,50 @@
+module Applications
+  class RevertToPending
+    REMOVEABLE_DECLARATION_STATES = %w[submitted voided ineligible].freeze
+
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attr_reader :application
+
+    attribute :change_status_to_pending
+    delegate :lead_provider_approval_status, to: :application
+
+    validates :change_status_to_pending, inclusion: { in: %w[yes] }
+    validates :lead_provider_approval_status, inclusion: { in: %w[accepted] }
+    validate :application_has_no_unremoveable_declarations
+
+    class << self
+      def call(application)
```

Comment:
We've been avoiding the `call` pattern in the new services up until now; I don't really feel strongly about it either way, but something the be aware of